packages/neon/neon/lib/src/widgets/account_tile.dart

```diff
@@ -47,54 +99,12 @@ class NeonAccountTile extends StatelessWidget {
           : null,
       leading: NeonUserAvatar(
         account: account,
-        showStatus: showStatus,
-      ),
-      title: ResultBuilder<ProvisioningApiUserDetails>.behaviorSubject(
-        stream: userDetailsBloc.userDetails,
-        builder: (final context, final userDetails) => Row(
-          children: [
-            if (userDetails.hasData) ...[
-              Flexible(
-                child: Text(
-                  userDetails.requireData.displayname,
-                  style: Theme.of(context).textTheme.bodyLarge!.copyWith(
-                        color: textColor,
-                      ),
-                  overflow: TextOverflow.ellipsis,
-                ),
-              ),
-            ],
-            if (userDetails.isLoading) ...[
-              const SizedBox(
-                width: 5,
-              ),
-              Expanded(
-                child: NeonLinearProgressIndicator(
-                  color: textColor,
-                ),
-              ),
-            ],
-            if (userDetails.hasError) ...[
-              const SizedBox(
-                width: 5,
-              ),
-              NeonException(
-                userDetails.error,
-                onlyIcon: true,
-                iconSize: 24,
-                onRetry: userDetailsBloc.refresh,
-              ),
-            ],
-          ],
-        ),
-      ),
-      subtitle: Text(
-        account.client.humanReadableID,
-        style: Theme.of(context).textTheme.bodySmall!.copyWith(
-              color: textColor,
-            ),
-        overflow: TextOverflow.ellipsis,
+        showStatus: showStatus && !showActive,
+        decoration: showActive && isActive ? const Icon(Icons.check_circle) : null,
```

Comment:
This check mark is quite huge. Please reduce the size to size of the status circle